package.json

```diff
@@ -69,12 +69,12 @@
     "@apollo/gateway": "2.1.1",
     "@apollo/subgraph": "2.1.1",
     "@apollo/utils.keyvadapter": "3.0.0",
-    "@aws-sdk/client-s3": "3.662.0",
-    "@aws-sdk/client-sqs": "3.662.0",
-    "@aws-sdk/credential-provider-node": "3.662.0",
-    "@aws-sdk/lib-storage": "3.662.0",
-    "@aws-sdk/s3-presigned-post": "3.662.0",
-    "@aws-sdk/s3-request-presigner": "3.662.0",
+    "@aws-sdk/client-s3": "3.689.0",
+    "@aws-sdk/client-sqs": "3.689.0",
+    "@aws-sdk/credential-provider-node": "3.687.0",
```

Comment:
Any specific reason why this is `687` while the other changes are `689`?

Reply:
No reason. I ran `yarn add $(jq -r '.dependencies | keys[]' package.json | grep '^@aws-sdk/')` to do the updates :shrug: 

Do you want the same version everywhere, or control which version we're using?

Reply:
Set the same `3.677.0` for all. `@aws-sdk/credentials-provider-node` doesn't have a newer version.